packages/theming/apple-theme/src/createAppleTheme.ios.ts

```diff
@@ -1,17 +1,38 @@
 import { ThemeReference } from '@fluentui-react-native/theme';
-import { Theme } from '@fluentui-react-native/theme-types';
-import { Appearance } from 'react-native';
+import { PartialTheme, Theme } from '@fluentui-react-native/theme-types';
+import { Appearance, NativeModules } from 'react-native';
 import { BaseAppleDarkThemeIOS, BaseAppleLightThemeIOS } from './appleTheme.ios';
 
+// Save this value between calls
+let partialThemeFromNativeModule: PartialTheme;
+
 export function createAppleTheme(): ThemeReference {
-  const themeRef = new ThemeReference({} as Theme, () => {
+  const baseAppleTheme = () => {
     const current = Appearance.getColorScheme();
     return current === 'light' ? BaseAppleLightThemeIOS : BaseAppleDarkThemeIOS;
-  });
+  };
+
+  const loadNativeTheme = (appleThemeReference: ThemeReference) => {
+    const module = NativeModules && NativeModules.MSFAppleThemeModule;
+    if (module) {
+      module.getApplePartialThemeWithCallback((error, applePartialTheme) => {
+        if (error) {
+          console.error(`Error retrieving apple theming module! ${error}`);
```

Comment:
Is there a reason you're using backquote instead of a regular single quote?

Reply:
that might just have been prettier when I decided to add the `${error}`. I don't like backticks and I'll see if I can just switch to single quotes and string concatenation. Good catch!